packages/react-component-library/src/components/AutocompleteE/AutocompleteEOption.tsx

```diff
@@ -3,11 +3,28 @@ import React from 'react'
 import { SelectBaseOption, SelectBaseOptionProps } from '../SelectBase'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AutocompleteEOptionProps extends SelectBaseOptionProps {}
+export interface AutocompleteEOptionProps extends SelectBaseOptionProps {
+  inputValue?: string
+}
+
+function emboldenInputValue(children: string, inputValue: string) {
+  if (inputValue) {
+    const regex = new RegExp(`(${inputValue})`, 'gi')
```

Comment:
Will need to first RegEx escape `inputValue` (for example typing `*` in the component breaks things at the moment) before putting it through the RegExp.

So something like (not tested...):

```tsx
import {escape as escapeHTML, escapeRegExp} from 'lodash'

const escapedInputValue = escapeRegExp(inputValue)

// use escapedInputValue in the RegExp

return escapeHTML(children.replace(regex, '<strong>$1</strong>'))
```